organization-bootstrap/business-units/README.md

```diff
@@ -0,0 +1,102 @@
+# Business-units based organizational sample
+
+This sample creates an organizational layout with two folder levels, where the first level is usually mapped to one business unit (infra, data, analythics) and the second level represents enviroments (prod, test). It also sets up all prerequisites for automation (GCS state buckets, service accounts, etc.), and the correct roles on those to enforce separation of duties at the environment level.
+
+This layout is well suited for small and medium-sized infrastructures managed by a smal set of teams, where the complexity in application resource ownership and access roles is mostly dealt with at the project level, and/or in the individual services (GKE, Cloud SQL, etc.). Its simplicity also makes it a good starting point for more complex or specialized layouts.
```

Comment:
Typo on `small`

Reply:
We should change this paragraph to more closely match this sample, and remove the duplication with the following paragraph.